cupyx/scipy/ndimage/_spline_prefilter_core.py

```diff
@@ -250,4 +250,6 @@ def get_raw_spline1d_kernel(axis, ndim, mode, order, index_type='int',
     # generate code handling batch operation of the 1d filter
     code += _batch_spline1d_strided_template.format(ndim=ndim, axis=axis,
                                                     block_size=block_size)
-    return cupy.RawKernel(code, 'cupyx_spline_filter')
+    kernel_name = (f'cupyx_scipy_ndimage_spline_filter_{ndim}d_ord{order}_'
+                   f'axis{axis}_{mode}')
+    return cupy.RawKernel(code, kernel_name)
```

Comment:
This is the most immediate error I can spot: line 212 should also be changed in order to fetch the correct kernel name:
https://github.com/cupy/cupy/blob/840e112e1dd5757eff074f1dc65dc6d950f34da5/cupyx/scipy/ndimage/_spline_prefilter_core.py#L212

Reply:
Yes, I have fixed this locally now, but other tests are still running. It was also necessary to `replace('-', '_')` in the mode name string to get a valid C++ function name for the kernel